src/hooks/auth.ts

```diff
@@ -49,10 +50,11 @@ export const useAuth = ({ middleware }: { middleware?: any } = {}) => {
     await api
       .post('/api/login', props)
       .then(async (resp) => {
-        {
-          console.log(resp.data, '--> resp.data');
-        }
         setAuthToken(resp.data);
+        setCookie(null, 'authToken', resp.data, {
```

Comment:
why is the first param null to setCookie?

Reply:
The first param `null` to cookies because no specific domain is set for the cookies. Passing `null`, the cookies will be associated with the current domain.